packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -56,6 +56,7 @@ export const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
             aria-label={messages.genre}
             placeholder={messages.genre}
             mount='parent'
+            // TODO: Use correct value for Genres based on label (see `convertGenreLabelToValue`)
             menu={{ items: GENRES }}
```

Comment:
@amendelsohn @dylanjeffers @Kyle-Shanks we're going to need to fix this for upload v2, i held off because it's not out yet and i didn't know how to specify an item where the label + value are different (seems to require an additional "elem" field to do that and i didn't know what to pass)

Reply:
made a ticket https://linear.app/audius/issue/C-2921/fix-value-passed-for-electronic-subgenres